### PR TITLE
Redirect non UK friends to the supporter landing page for tier change

### PR DESCRIPTION
**Pull request by AWare**
Redirect non UK friends to the the supporter page when they try and change tier- as there's no partner/patron tier for them to upgrade to.

### CODE REVIEW

File: frontend/app/controllers/TierController.scala [hunk 1]
```diff
@@ -61,10 +61,17 @@ object TierController extends Controller with ActivityTracking
...
 
-  def change() = SubscriptionAction { implicit request =>
+  def change() = SubscriptionAction.async { implicit request =>
     implicit val countryGroup = UK
     implicit val c = catalog
-    Ok(views.html.tier.change(request.subscriber.subscription.plan.tier, catalog))
+    for {
+      cg <- request.getIdentityCountryGroup
+    } yield {
+      if (cg.exists(_ != CountryGroup.UK) && request.paidOrFreeSubscriber.isLeft)
```
Comment (philwills) [inline frontend/app/controllers/TierController.scala]: `isLeft` doesn't really feel very descriptive here.

Are we using a disjunction with paid details left and free right?

Comment (AWare) [inline frontend/app/controllers/TierController.scala]: Free on the left, paid on the right. That could do with being clearer.

**Comment by paulbrown1982** [comment]
👍

**Comment by AWare** [comment]
Testerd

**Pull request merged**